tests/validation/tests/v3_api/common.py

```diff
@@ -2084,15 +2084,29 @@ def readDataFile(data_dir, name):
         return f.read()
 
 
-def set_url_password_token(rancher_url, server_url=None):
+def set_url_password_token(rancher_url, server_url=None, version="oldUI"):
     """Returns a ManagementContext for the default global admin user."""
     auth_url = \
         rancher_url + "/v3-public/localproviders/local?action=login"
-    r = requests.post(auth_url, json={
-        'username': 'admin',
-        'password': 'admin',
-        'responseType': 'json',
-    }, verify=False)
+    rpassword = 'admin'
+    print(auth_url)
+    if version.find("master") > -1 or version.find("2.6") > -1:
```

Comment:
Would be good to use `rancher_version_check()` from [this open PR](https://github.com/rancher/rancher/pull/34620) when comparing rancher versions.